tests/kernel/arm_irq_vector_table/src/arm_irq_vector_table.c

```diff
@@ -64,14 +55,23 @@ void isr2(void)
 }
 
 /**
- * @brief test installation of ISRs directly in the vector table
+ * @defgroup kernel_interrupt Interrupt_tests
```

Comment:
should add a suffix (_tests) to the group to differentiate this from the actual feature documentation, so this becomes kernel_interrupt_tests

Reply:
should be 

`@defgroup kernel_interrupt Interrupts`

first argument is the identifier, the second is the title in the documentation